src/lib.rs

```diff
@@ -19,7 +19,10 @@
 mod arc_list;
 mod atomic_waker;
 mod delay;
-mod global;
+#[cfg(not(all(target_arch = "wasm32", target_os = "unknown")))]
+mod global_native;
+#[cfg(all(target_arch = "wasm32", target_os = "unknown"))]
```

Comment:
We cannot assume that this is "browser wasm". We should feature gate this on `wasm_bindgen` instead.

Reply:
We could just use `target_arch = "wasm32"`, but I'd rather not make the wasm support optional if possible.

Reply:
To clarify: I'd rather not make everyone who wants to use this crate with wasm support to use `futures-timer = { version = "...", features = ["wasm-bindgen"] }`. While it would be explicit, imo things should _just work_.

Reply:
Changed my mind :^)